content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -261,6 +261,13 @@ indicates one of the following reasons for the Pod termination:
 `TerminationByKubelet`
 : Pod has been terminated by the kubelet, because of either {{<glossary_tooltip term_id="node-pressure-eviction" text="node pressure eviction">}} or the [graceful node shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown).
 
+{{< note >}}
```

Comment:
There are already 2 existing Notes mentioned under this section.
Again mentioning Note Right after one is not good at least to me.

Reply:
You can merge both the notes with bullets, which may be more feasible here.

Reply:
makes sense, we could maybe also just add the new text below the list without a note tag? Avoiding to touch the existing node

Reply:
Tentatively I pushed a change to drop the note, but I'm also ok to combine both paragraphs into a single note. Let me know what you think.

Reply:
> makes sense, we could maybe also just add the new text below the list without a note tag? Avoiding to touch the existing node

This would be fine.

Reply:
Thx